libraries/apollo-api/build.gradle.kts

```diff
@@ -16,6 +16,12 @@ kotlin {
       }
     }
 
+    findByName("jvmMain")?.apply {
+      dependencies {
+        compileOnly(libs.guava.jre)
+      }
+    }
```

Comment:
I have mixed feelings about pulling guava in `apollo-api`, even if it's compileOnly. `assertOneOf` isn't that huge. Could we generate it with kotlin poet instead?

Reply:
Or maybe just write it verbatim, no need for KotliPoet actually. Get a template and copy it to `packageName.utils.Assertions.kt`?

Reply:
Done in f5efb16aa992d6cb831d408fa21636349b9a0044